.github/workflows/SyncToAtlas.yml

```diff
@@ -0,0 +1,54 @@
+name: Sync Master to Atlas Repos
+
+on: 
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  atlas:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master 
+    - name: Add O'Reilly remotes
+      run: |
+        git remote add oreilly git@git.atlas.oreilly.com:lenucksi/learningpath-introduction.git
```

Comment:
Is this some fork of the actual learning path?

Reply:
Thanks.  Will anyone else ever need to see what is in _Atlas_?  It looks like I, myself, can't view this repo, for example.

Reply:
> Thanks. Will anyone else ever need to see what is in _Atlas_? 

Aside from people at O'Reilly? No. Some of it is likely O'Reilly IP anyway.

> It looks like I, myself, can't view this repo, for example.

Now you should be able to.

Reply:
> Thanks. Will anyone else ever need to see what is in _Atlas_? 

Aside from people at O'Reilly? No. Some of it is likely O'Reilly IP anyway.

> It looks like I, myself, can't view this repo, for example.

Now you should be able to.